site/en/docs/privacy-sandbox/unified-origin-trial/index.md

```diff
@@ -30,6 +30,12 @@ testing against the APIs.
 
 ## Check the status of the origin trial {: #status}
 
+### October 2022
+
+-  Origin trial [extended to Chrome 110](https://groups.google.com/a/chromium.org/g/blink-dev/c/xm9EvnaVBj8). 
+[Getting started with Chrome's origin trials](/docs/web-platform/origin-trials/#renew) 
+explains how to renew origin trial enrollment.
+
```

Comment:
Could you include the table with the API status in here too.

Reply:
Done!